commit ebb56bdb5cb14d96cc8f917a2474d8c109b4b525
Author: Sakari Ailus <sakari.ailus@linux.intel.com>
Date:   Mon Jan 2 08:32:47 2017 -0200

    media: Properly pass through media entity types in entity enumeration
    
    commit 98d85f3cb912fde14593ead54dea4c1a00b3966f upstream.
    
    When the functions replaced media entity types, the range which was
    allowed for the types was incorrect. This meant that media entity types
    for specific devices were not passed correctly to the userspace through
    MEDIA_IOC_ENUM_ENTITIES. Fix it.
    
    Fixes: commit b2cd27448b33 ("[media] media-device: map new functions into old types for legacy API")
    Reported-and-tested-by: Antti Laakso <antti.laakso@intel.com>
    
    Signed-off-by: Sakari Ailus <sakari.ailus@linux.intel.com>
    Acked-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
    Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1226_ebb56bdb5cb14d96cc8f917a2474d8c109b4b525.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1226_ebb56bdb5cb14d96cc8f917a2474d8c109b4b525.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
